forge-gui/res/cardsfolder/upcoming/faramir_prince_of_ithilien.txt

```diff
@@ -0,0 +1,15 @@
+Name:Faramir, Prince of Ithilien
+ManaCost:2 W U
+Types:Legendary Creature Human Noble
+PT:3/3
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigChooseOpp | TriggerDescription$ At the beginning of your end step, choose an opponent. At the beginning of that player's next end step, you draw a card if they didn't attack you that turn. Otherwise, create three 1/1 white Human Soldier creature tokens.
+SVar:TrigChooseOpp:DB$ ChoosePlayer | Defined$ You | Choices$ Opponent | SubAbility$ DBDelayTrig
+SVar:DBDelayTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | ValidPlayer$ Player.IsTriggerRemembered | RememberObjects$ ChosenPlayer | Execute$ TrigBranch | TriggerDescription$ At the beginning of that player's next end step, you draw a card if they didn't attack you that turn. Otherwise, create three 1/1 white Human Soldier creature tokens.
+SVar:TrigBranch:DB$ Branch | BranchConditionSVar$ X | BranchConditionSVarCompare$ GE1 | TrueSubAbility$ DBToken | FalseSubAbility$ DBDraw | SubAbility$ DBCleanup
+SVar:DBDraw:DB$ Draw
+SVar:DBToken:DB$ Token | TokenScript$ w_1_1_human_soldier | TokenAmount$ 3 
+SVar:DBCleanup:DB$ Cleanup | ClearChosenPlayer$ True
+SVar:X:PlayerCountDefinedChosenPlayer$HasPropertyattackedYouTheirCurrentTurn
```

Comment:
Try `PlayerCountDefinedTriggeredPlayer`

Reply:
are you testing from this branch? because the support commit isn't in that yet

Reply:
oh good point. It's not available yet from upstream on my end

Reply:
Seems to work now thanks